conf/defaults.ini

```diff
@@ -738,8 +738,9 @@ global_alert_rule = -1
 
 #################################### Unified Alerting ####################
 [unified_alerting]
-# Enable the Unified Alerting sub-system and interface. When enabled we'll migrate all of your alert rules and notification channels to the new system. New alert rules will be created and your notification channels will be converted into an Alertmanager configuration. Previous data is preserved to enable backwards compatibility but new data is removed.
-enabled = false
+# Enable the Unified Alerting sub-system and interface. When enabled we'll migrate all of your alert rules and notification channels to the new system. New alert rules will be created and your notification channels will be converted into an Alertmanager configuration. Previous data is preserved to enable backwards compatibility but new data is removed. If it is not defined the actual state would be defined in runtime. When this configuration section and flag are not defined, the state is defined at runtime. See the documentation for more details.
+# Caution! Once the flag is enabled and a user creates alert rules, the disabling of the flag will cause all the rules to be deleted when it is enabled the next time.
```

Comment:
```suggestion
# Enable the Unified Alerting sub-system and interface. When enabled we'll migrate all of your alert rules and notification channels to the new system. New alert rules will be created and your notification channels will be converted into an Alertmanager configuration. Previous data is preserved to enable backwards compatibility but new data is removed when switching. When this configuration section and flag are not defined, the state is defined at runtime. See the documentation for more details.
```

We already had a warning on data being removed so I have removed the caution notice - should we also update the documentation so that it matches this test and states that the default value is `not defined by the configuration`.